src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -95,6 +144,7 @@ public void FillFromJson(JsonElement item, string project, IDeveloperId? devId)
             State = item.GetProperty("powerState").ToString();
             CPU = item.GetProperty("hardwareProfile").GetProperty("vCPUs").ToString();
             Memory = item.GetProperty("hardwareProfile").GetProperty("memoryGB").ToString();
+            DiskSize = item.GetProperty("storageProfile").GetProperty("osDisk").GetProperty("diskSizeGB").ToString();

```

Comment:
Is it possible one of GetProperty returns null or throws an exception? Is it Ok we'd skip all other properties and exit here with an exception?

Reply:
Unless the schema is changed, this cannot happen. But I have updated it to only fully fail if the most important properties fail.